shared/utils.py

```diff
@@ -40,24 +40,40 @@ def wrapper(*args, **kwargs):
 
 
 @need_connection
-def get_sld_id(object_id: int):
+def get_object_module(object_id: int, object_class: int = None):
```

Comment:
Shouldn't this have a comment explaining args and return value rather than `_get_module_object()` given that `_get_module_object()` appears to be private?